tests/performance/pom.xml

```diff
@@ -150,6 +151,76 @@
             </build>
         </profile>
         <profile>
+            <id>micro-benchmark</id>
+            <build>
+                <plugins>
+                    <plugin>
```

Comment:
Its a new profile to run the jmh tests as it was from the compiled jar and then upload the output to elastisearch.